phpBB/memberlist.php

```diff
@@ -1708,7 +1711,7 @@
 			$id_cache = array();
 			while ($row = $db->sql_fetchrow($result))
 			{
-				$row['session_time'] = (!empty($session_times[$row['user_id']])) ? $session_times[$row['user_id']] : 0;
+				$row = array_merge($row, !empty($session_ary[$row['user_id']]) ? $session_ary[$row['user_id']] : ['session_time' => 0, 'session_viewonline' => 0]);
```

Comment:
The readability of this line is not that great and running that many array_merge() calls is also not recommended. I'd go with this instead:
```
				$row['session_time'] = $session_ary[$row['user_id']]['session_time'] ?? 0;
				$row['session_viewonline'] = $session_ary[$row['user_id']]['session_viewonline'] ?? 0;
```

Reply:
Done.

Thanks 👍